style.css

```diff
@@ -224,10 +224,16 @@ hr {
 	grid-row: 1;
 }
 
+.logo{
```

Comment:
This is the logo class that is used to keep the logo on the right side of the page